console/console-init/ui/src/modules/address-space/dialogs/CreateMessagingProject/MessagingProjectConfiguration.tsx

```diff
@@ -160,63 +153,41 @@ export const Configuration: React.FunctionComponent<IConfiguration> = ({
       authenticationServices.addressSpaceSchema_v2.forEach(authService => {
         if (authService.metadata.name === type) {
           authenticationServiceOptions = authService.spec.authenticationServices.map(
-            service => {
-              return {
-                value: service,
-                label: service
-              };
-            }
+            service => ({
+              value: service,
+              label: service,
+              key: service
+            })
           );
         }
       });
     }
     return authenticationServiceOptions;
   };
 
-  const handleBrokeredChange = () => {
-    setIsBrokeredChecked(true);
-    setIsStandardChecked(false);
-    setPlan(" ");
-    setAuthenticationService(" ");
-    setType("brokered");
-  };
-
-  const handleStandardChange = () => {
-    setIsStandardChecked(true);
-    setIsBrokeredChecked(false);
-    setPlan(" ");
-    setAuthenticationService(" ");
-    setType("standard");
-  };
-
-  const handleNameChange = (name: string) => {
-    setName(name);
-    !dnsSubDomainRfc1123NameRegexp.test(name)
-      ? setIsNameValid(false)
-      : setIsNameValid(true);
-  };
-
   return (
-    <>
-      <AddressSpaceConfiguration
-        onNameSpaceSelect={onNameSpaceSelect}
-        handleNameChange={handleNameChange}
-        handleBrokeredChange={handleBrokeredChange}
-        onPlanSelect={onPlanSelect}
-        handleStandardChange={handleStandardChange}
-        onAuthenticationServiceSelect={onAuthenticationServiceSelect}
-        namespace={namespace}
-        namespaceOptions={getNameSpaceOptions()}
-        name={name}
-        isNameValid={isNameValid}
-        isStandardChecked={isStandardChecked}
-        isBrokeredChecked={isBrokeredChecked}
-        type={type}
-        plan={plan}
-        planOptions={getPlanOptions()}
-        authenticationService={authenticationService}
-        authenticationServiceOptions={getAuthenticationServiceOptions()}
-      />
-    </>
+    <AddressSpaceConfiguration
+      onNameSpaceSelect={onNameSpaceSelect}
+      handleNameChange={handleNameChange}
+      handleBrokeredChange={handleBrokeredChange}
+      onPlanSelect={onPlanSelect}
+      handleStandardChange={handleStandardChange}
+      onAuthenticationServiceSelect={onAuthenticationServiceSelect}
+      namespace={namespace || ""}
+      namespaceOptions={getNameSpaceOptions()}
+      name={name || ""}
+      isNameValid={isNameValid || false}
+      isStandardChecked={type === "standard"}
```

Comment:
why do we need separate two checks variable for radio options. It should be handle by single varibale by passing the value of selected radion option. It should be either "brokered" or "standard".
Can you please try to handle in single variable.